samples/snippets/src/test/java/com/example/bigquery/CreateExternalTableAwsIT.java

```diff
@@ -120,18 +90,19 @@ public void tearDown() throws IOException {
 
   @Test
   public void testCreateExternalTableAws() {
-    String sourceUri = "s3://cloud-samples-tests/us-states.csv";
+    String sourceUri = "s3://steph-omni-test-bucket/us-states.csv";
```

Comment:
Might not be your best final choice for a bucket name.

Reply:
Yes - you're right. Created a new bucket for devrel sampels testing. Thank you!